editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -312,9 +311,7 @@ function useStartCanvasSession(): (
       }
 
       dispatch([
-        CanvasActions.createDragState(
-          startNewSelectModeCanvasSession(start, { type: 'BOUNDING_AREA' }),
-        ),
+        // TODO actually dispatch a canvas interaction session
```

Comment:
If we're making the strategies PR zero based then the whole of `useStartCanvasSession` should be deleted

Reply:
fair point

Reply:
done! I actually removed the feature flag entirely